plugins/auth-backend/src/identity/TokenFactory.ts

```diff
@@ -221,6 +220,13 @@ export class TokenFactory implements TokenIssuer {
       );
     }
 
+    // Store the user info in the database upon successful token
+    // issuance so that it can be retrieved later by limited user tokens
+    await this.userInfoDatabaseHandler.addUserInfo({
+      userEntityRef: sub,
+      ownershipEntityRefs: ent,
```

Comment:
So related to my comment elsewhere - i wonder if this can't just be `({ claims })` instead. Well, we may perhaps denylist some known unnecessary fields, like `exp`, `iat`, `iss`, `aud` and `uip`, from the body itself, but keep whatever custom stuff remains after that in addition to `sub` and `ent`.

And this is just the internal storage model; there's nothing stopping us from translating that to something else in the JSON response from the service itself.